packages/web-app-epub-reader/src/App.vue

```diff
@@ -1,14 +1,18 @@
 <template>
-  <div class="epub-reader oc-flex oc-p-s">
-    <oc-list class="epub-reader-chapters-list oc-visible@l">
+  <div class="epub-reader oc-flex">
+    <oc-list class="epub-reader-chapters-list oc-pl-s">
```

Comment:
Why did you decide to remove the `oc-visible@l`? With that the chapter nav sidebar is quite distorted on smaller screens, see screenshots.
<img width="881" alt="Screenshot 2024-02-13 at 22 12 08" src="https://github.com/owncloud/web/assets/3532843/c39e3926-b067-4cea-91d2-b12f3916de39">
<img width="858" alt="Screenshot 2024-02-13 at 22 12 19" src="https://github.com/owncloud/web/assets/3532843/42aca6ac-3e10-46b3-b79d-94fa1bc4fb17">
<img width="544" alt="Screenshot 2024-02-13 at 22 12 32" src="https://github.com/owncloud/web/assets/3532843/91dfdb95-871a-4a08-b9b5-7ed34ad8fb99">

Reply:
Hm yes. I think that's something I need to sort out when solving the iPad optimization task 🤔 
But we can bring that back in the meantime.